__tests__/userlist.test.js

```diff
@@ -0,0 +1,29 @@
+const integration = "userlist";
+const name = "Userlist";
+
+const fs = require("fs");
+const path = require("path");
+const version = "v0";
+
+const transformer = require(`../${version}/destinations/${integration}/transform`);
+
+const inputDataFile = fs.readFileSync(
+  path.resolve(__dirname, `./data/${integration}_input.json`)
+);
+const outputDataFile = fs.readFileSync(
+  path.resolve(__dirname, `./data/${integration}_output.json`)
+);
+
+const inputData = JSON.parse(inputDataFile);
+const expectedData = JSON.parse(outputDataFile);
+
+inputData.forEach((input, index) => {
+  it(`${name} Tests: payload: ${index}`, async () => {
```

Comment:
Please remove the `async/await` as the `transformer.process` method is not `async`.

Reply:
Ok, removed. FYI: This pattern is common in other tests.

Reply:
Yeah. We're in the process of the code cleanups for this.